packages/core/src/components/Card/index.tsx

```diff
@@ -8,17 +8,31 @@ export { Content };
 export type CardProps = {
   /** List of `Content`s blocks to contain content. */
   children: NonNullable<React.ReactNode>;
+  /** Hide shadow. */
+  noShadow?: boolean;
   /** Set overflow to be visible. */
   overflow?: boolean;
+  /** Whether the card is selected or not. */
+  selected?: boolean;
   /** Custom style sheet. */
   styleSheet?: StyleSheet;
 };
 
 /**
  * An abstract layout to use as a base for cards.
  */
-export default function Card({ children, overflow, styleSheet }: CardProps) {
+export default function Card({ children, noShadow, overflow, selected, styleSheet }: CardProps) {
   const [styles, cx] = useStyles(styleSheet ?? styleSheetCard);
 
-  return <div className={cx(styles.card, overflow && styles.card_overflow)}>{children}</div>;
+  return (
+    <div
+      className={cx(styles.card,
```

Comment:
@monicapharm I think prettier is still failing in CI, I think because of this line